### ISSUE TITLE
Improve keyboard accessibility

### ISSUE THREAD

**Issue by craigcook** [skycatchfire/stateofmozilla#2]
Just some things I've noticed while trying to navigate with a keyboard, in no particular order. I may add more things to the list as I find them.

- Articles that open in modals don't receive focus, so when tabbing through links the tab order stays where it was on the content behind the modal.
- Article modals aren't scrollable with the keyboard (space, arrows, or page up/down). Setting focus to the modal should fix this, too.
- ESC should close/dismiss a modal and return focus to the point of origin.
- Focus styles could be improved. Right now it's just the default focus ring, which is better than nothing, but can be hard to see in come cases (like on black buttons). Often just matching :hover style on links and buttons (in addition to the focus ring) is good enough.

**Comment by craigcook** [comment]
Well, ESC does close the modals now, but the other things mentioned in this issue are still present, especially the lack of focus setting. I don't think this should have been closed yet but I don't have permissions to reopen it. I can file separate issues for setting focus and focus styling, if that's preferable.
